airgun/views/computeresource.py

```diff
@@ -151,22 +161,31 @@ def is_displayed(self):
             self.breadcrumb, exception=False)
         return (
             breadcrumb_loaded
-            and self.breadcrumb.locations[0] == 'Compute resources'
+            and self.breadcrumb.locations[0] == 'Compute Resources'
             and self.breadcrumb.read().startswith('Edit ')
         )
 
 
+class ComputeResourceTabTable(SatTable):
```

Comment:
`SatTable` is deprecated now according to https://github.com/SatelliteQE/airgun/pull/253/files. Could you try to use `Table` widget directly?

Reply:
Ok will test that with Table

Reply:
With table having this result when no rows found
```
[{'Name': 'Suggestions Check the javascript regular expression syntax of the search term. Check that the correct menu option is chosen (token ID vs. user ID). Use wildcards (* to match 0 or more characters, + to match 1 or more characters, ? to match 0 or 1 character). Clear the search field, then click Search to return to the 20 most recent records.', 'Operating System': 'Suggestions Check the javascript regular expression syntax of the search term. Check that the correct menu option is chosen (token ID vs. user ID). Use wildcards (* to match 0 or more characters, + to match 1 or more characters, ? to match 0 or 1 character). Clear the search field, then click Search to return to the 20 most recent records.', 'Username': 'Suggestions Check the javascript regular expression syntax of the search term. Check that the correct menu option is chosen (token ID vs. user ID). Use wildcards (* to match 0 or more characters, + to match 1 or more characters, ? to match 0 or 1 character). Clear the search field, then click Search to return to the 20 most recent records.', 'UUID': 'Suggestions Check the javascript regular expression syntax of the search term. Check that the correct menu option is chosen (token ID vs. user ID). Use wildcards (* to match 0 or more characters, + to match 1 or more characters, ? to match 0 or 1 character). Clear the search field, then click Search to return to the 20 most recent records.', 'User data': 'Suggestions Check the javascript regular expression syntax of the search term. Check that the correct menu option is chosen (token ID vs. user ID). Use wildcards (* to match 0 or more characters, + to match 1 or more characters, ? to match 0 or 1 character). Clear the search field, then click Search to return to the 20 most recent records.', 'Actions': 'Suggestions Check the javascript regular expression syntax of the search term. Check that the correct menu option is chosen (token ID vs. user ID). Use wildcards (* to match 0 or more characters, + to match 1 or more characters, ? to match 0 or 1 character). Clear the search field, then click Search to return to the 20 most recent records.'}]
```
![screenshot from 2018-12-10 19-02-07](https://user-images.githubusercontent.com/131553/49748426-4d86f780-fcae-11e8-9233-b9dd1b06875c.png)

Reply:
are we deprecating a working scenario in place of not working one?

Reply:
yeah, we are moving to use solely `Table` in all places, so you can verify whether your `Name` is not equal to expected (when you delete entity for example) or that part of expected message (because it is too long) is present in `Name`

Reply:
class completely deleted and Table implemented